coordinator/manifest/manifest.go

```diff
@@ -457,14 +461,37 @@ func EncodeSecretDataToBase64(data interface{}) (string, error) {
 	return base64.StdEncoding.EncodeToString([]byte(raw)), nil
 }
 
-// ManifestTemplateFuncMap defines the functions which can be specified for secrets in the in go template format
-var ManifestTemplateFuncMap = template.FuncMap{
+// EncodeEnvSecretDataToRaw encodes only secrets of type plain to raw byte string
+func EncodeEnvSecretDataToRaw(data interface{}) (string, error) {
```

Comment:
I don't quite like to re-use the type `raw` here but impose limits onto it when we don't do it for Files. This might be unexpected.

Reply:
What about a more explicit type like `env` or smth.?

Reply:
The problem with `env` is that when you can also inject stuff like base64, what is `env` supposed to represent? I would just go with `(c)string` maybe.

Reply:
Ok, let's move on here. I'm fine with `string`, if Daniel agrees let's just use it:-)

Reply:
string is OK.
As long as we make it clear in the documentation which tags do what, there shouldn't be any problems